src/Lumper.UI/Converters/TaskStatusConverter.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Globalization;
+using Avalonia.Data;
+using Avalonia.Data.Converters;
+using Lumper.UI.ViewModels.Tasks;
+
+namespace Lumper.UI.Converters;
+
+/// <summary>
+///     format TaskStatus
+/// </summary>
+public class TaskStatusConverter : IValueConverter
+{
+    public static TaskStatusConverter Instance
+    {
+        get;
+    } = new();
+
+    public object Convert(object? value, Type targetType, object? parameter,
+        CultureInfo culture)
+    {
+        if (value is TaskStatus status)
+        {
+            switch (status)
+            {
+                case TaskStatus.Waiting:
+                    return "|";
+                case TaskStatus.Running:
+                    return "➠";
+                case TaskStatus.Success:
+                    return "✓";
+                case TaskStatus.Failed:
+                    return "⚠";
+                case TaskStatus.Unknown:
+                default:
+                    return " ";
+            }
+        }
+        return new BindingNotification(
+            new ArgumentOutOfRangeException(nameof(value)),
+            BindingErrorType.DataValidationError);
```

Comment:
Just return the notif if `value is not TaskStatus status`

Reply:
if I return early I have to cast again .. I like this more .. or what do you mean? `is *not* something *value*` doesn't make sense, does it?

Reply:
Try it, compiler makes the inference fine! i.e.
```c#
if (value is not TaskStatus status)
    return new BindingNotification(
        new ArgumentOutOfRangeException(nameof(value)),
        BindingErrorType.DataValidationError);
switch (status)
{
    case TaskStatus.Waiting:
        return "|";
    case TaskStatus.Running:
        return "➠";
    case TaskStatus.Success:
        return "✓";
    case TaskStatus.Failed:
        return "⚠";
    case TaskStatus.Unknown:
    default:
        return " ";
}
```